docs/reference/feature-servers/python-feature-server.md

```diff
@@ -152,8 +152,8 @@ curl -X POST \
   }' | jq
 ```
 
-### Pushing features to the online store
-You can push data corresponding to a push source to the online store (note that timestamps need to be strings):
+### Pushing features to the online and offline stores.
+You can push data corresponding to a push source to the online store and offline stores (note that timestamps need to be strings):
```

Comment:
nit: either `online and offline stores` or `online store and offline store`

Reply:
fixed

Reply:
Hm wonder why the commit didn't pick that up, fixed thist ime.